contracts/base/Loans.sol

```diff
@@ -41,26 +41,50 @@ contract Loans is LoansInterface, SignerRole {
     // At any time, this variable stores the next available loan ID
     uint256 public loanIDCounter;
 
+    uint256 public timeWindowToTakeOutLoan = 7 days;
+
     PairAggregatorInterface public priceOracle;
+
     LendingPoolInterface public lendingPool;
 
+    LoanTermsConsensusInterface public loanTermsConsensus;
```

Comment:
I think that can just be stored as an address because the only place it is used is in a modifier. The actual interface is never needed